lua/wikis/commons/MvpTable.lua

```diff
@@ -121,13 +129,7 @@ end
 ---@param args mvpTableParsedArgs
 ---@return string
 function MvpTable._buildConditions(args)
```

Comment:
For cleaner code, fetch all match ids from the tournament, and then do `MatchGroupUtil.fetchMatches`